gen_scriptdriver_script.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+java -cp lib/newrelic-unix-monitor.jar com.newrelic.infra.unix.ScriptDriver
```

Comment:
This file is currently at the root of the project. Does it make sense to move into the`scripts` directory?

Reply:
This is a driver file . Yes! t has to be at root .